drivers/serial/uart_realtek_rts5912.c

```diff
@@ -31,6 +34,76 @@ struct uart_rts5912_device_config {
 struct uart_rts5912_dev_data {
 };
 
+DT_INST_FOREACH_STATUS_OKAY(PINCTRL_DT_INST_DEFINE);
+#if defined(CONFIG_PM)
+#include <reg/reg_gpio.h>
+#include "zephyr/drivers/gpio/gpio_rts5912.h"
+uint32_t rx_wakeup_pin_num;
+uint32_t *rts5912_rx_wake_reg;
+struct gpio_dt_spec uart_rx_wakeup = GPIO_DT_SPEC_GET(DT_NODELABEL(uart0_wrapper), gpios);
```

Comment:
Could you please elaborate why is this hard-coded and not sourced from DT? Thanks